src/fsearch_window_actions.c

```diff
@@ -273,6 +273,21 @@ fsearch_window_action_copy (GSimpleAction *action,
 }
 
 static void
+fsearch_window_action_copy_filepath (GSimpleAction *action,
+                                     GVariant      *variant,
+                                     gpointer       user_data)
+{
+    FsearchApplicationWindow *self = user_data;
+    GtkTreeSelection *selection = fsearch_application_window_get_listview_selection (self);
+    if (selection) {
+        GList *file_list = NULL;
+        gtk_tree_selection_selected_foreach (selection, copy_file, &file_list);
+        file_list = g_list_reverse (file_list);
+        clipboard_copy_filepath_list (file_list);
```

Comment:
That's an implicit declaration. `clipboard_copy_filepath_list` isn't exported in `clipboard.h`.